principles.md

```diff
@@ -0,0 +1,14 @@
+# Principles made during working session
+
+1. To achieve relevant industrial impact, the projects need to bridge the gab between research and product development
+2. Align the education with industry & academic needs by teaching fundamental theories through practical applications to ensure future og industry & academia is secured for innovative real-world solutions
+3. People over profits - To begin your study research, you need a strong motivation: Make sure you choose a path og (personal) growth, where you can also make some money
```

Comment:
As far as I remember, the focus of this one was more in the direction of using science and knowledge for the good. I.e. to steer research in the direction of benefitting humans, instead of merely focusing on optimizing for making more money. Analog to not developing nuclear weapons for war, but for humanity. 

Maybe we could express that best like this:

```suggestion
3. People over profits - To begin your study research, you need a strong motivation: Make sure you choose a path that may benefit humans, where you can also make some money
```

Or maybe something like `...a path that may benefit all humans...`,  or `...a path that may benefit humanity...`, etc.